Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js

```diff
@@ -168,7 +174,6 @@
             config.buttonId = Ext.id();
             config = Ext.applyIf(config||{}, {
                 html: this.graphTemplate.apply(config),
-                maxWidth: 800,
```

Comment:
This makes the component graphs look terrible, can we explicitly set this to null on the pop up graph? 
![screenshot 2015-06-11 07 28 07](https://cloud.githubusercontent.com/assets/383463/8106971/76b76754-100b-11e5-9e5c-7b0534b53da0.png)